run.sh

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env sh
+# Convenience script to run the wikiparser with the maps generator as a drop-in replacement for the scraper.
+#
+# 1. Finds latest map build.
+# 2. Builds wikiparser.
+# 3. Extracts wikidata ids and wikipedia urls from generator intermediate files `id_to_wikidata.csv` and `wiki_urls.txt`
+# 4. Runs wikiparser for all input dump files.
+#
+# For more information on the map generator, see
+# <https://github.com/organicmaps/organicmaps/blob/b52b42bd746fdb8daf05cc048f0b22654cfb9b8e/tools/python/maps_generator/README.md>.
+
+USAGE="./run.sh DUMP_FILE.json.tar.gz [DUMP_FILE.json.tar.gz...]
+
+Environment Variables:
+- MAPS_BUILD_ROOT: The directory with all maps builds, defaults to $HOME/maps_build, ignored if MAPS_DIR is set.
+- MAPS_DIR: The map build directory to use for generating descriptions, defaults to the latest directory in MAPS_BUILD_ROOT."
+
+set -eu
```

Comment:
```suggestion
set -euxo pipefail
```
Checking pipe failures helps.
If -x echo doesn't hurt, then it can be always used, for better understanding what magic is going under the hood.

Reply:
`pipefail` is unique to bash, I wrote this as a posix sh script. Happy to switch if you'd rather use bash.

Reply:
Bash is the default shell used in [many companies](https://google.github.io/styleguide/shellguide.html). It is a good practice to write bash scripts and use bash features.

`#!/usr/bin/env bash`